src/component/dashboard/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import uuid from 'uuid/v1';
+import NoteForm from './../note-form/index';
+// import NoteItem from './../note-list/note-item/index';
+import autoBind from '../../utils/index';
+
+export default class Dashboard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      notes: [],
+      error: null,
+    };
+
+    autoBind.call(this, Dashboard);
+  }
+
+  handleAddNote(note) {
+    if (note.title === '') {
+      return this.setState({ error: true });
+    }
+
+    note.createdOn = new Date();
+    note.id = uuid();
+
+    return this.setState((previousState) => {
+      return {
+        notes: [...previousState.notes, note],
+        error: null,
+      };
+    });
+  }
+
+  // handleRemoveNote() {
```

Comment:
Hint: pass in a note id parameter here
`.filter()` through your array of notes where `note.id !== id`